netlify/util/github-utils.ts

```diff
@@ -195,10 +195,11 @@ async function getSubmittedFindingsFromFolder(
           })
           .map((f) => {
             const [key, ext] = f.name.split(".");
-            const [handle, issueNumber] = key.split("-");
+            const splitFileName = key.split("-");
+            const issueNumber = splitFileName.pop();
```

Comment:
This is a small function so it's easy to connect the dots, but I would rather see handle explicitly assigned here too and then never use `splitFileName` again after that, since already after this pop it's no longer actually the  split file name anymore. (I sometimes use an underscore naming convention like `_splitFileName` to indicate sort of a "temporary"/"messy" variable like this one but dunno how common or following any code standard that is.)

Reply:
Thanks for the feedback! Implemented.